docs/reference/runtime/Bootstrap.md

```diff
@@ -1,12 +1,18 @@
 ## Bootstrap
 
-From a reset state, you can expect the following hooks and conditions. You can find more information in [`mbed_boot.c`](https://github.com/ARMmbed/mbed-os/blob/7b428916f5b35da28a1b18e2c149c1115bb89d92/rtos/TARGET_CORTEX/mbed_boot.c).
```

Comment:
While the function calls are not ideal here the concept of RAM initialization, clock initialization, CTORs etc seems important.

Reply:
Is it important from user point of view? I thought to document it in the porting guide, but from user's point of view it's all done by when the execution reaches main or mbed_main

Reply:
The specifics of the calls are not important but the concept of how the system works is important.